src/adv.py

```diff
@@ -65,31 +65,54 @@
 #
 # If the user enters "q", quit the game.
 
+
+def try_direction(direction):
+    attribute = direction[0].lower() + "_to"
+
+    if hasattr(player.current_room, attribute):
+        return getattr(player.current_room, attribute)
+    else:
+        print("There's nothing there!")
+        return player.current_room
+
+
 print(tw.dedent(
-    f"""\
+    f"""
     You are in room: {player.current_room.name}
-    {player.current_room.description}!\
-    """))
-
-print(tw.dedent('''
-        What would you like to do?
-        [n] Go North [e] Go East [s] Go South [w] Go West
-        [l] Look around
-        [get <item>] or [take <item>] Put <item> in your inventory
-        [drop <item>] Remove item from your inventory
-        [i] Show inventory
-        [q] Quit
-    '''))
+    {player.current_room.description}!
+"""))
+
+help_str = tw.dedent('''\
+    What would you like to do?
+    [check surroundings] Look around
+    [check inventory] Show inventory
+    [get <item>] or [take <item>] Put <item> in your inventory
+    [drop <item>] Remove item from your inventory
+    [<go, move> <n, e, s, w>] Go North, East, South, or West
+    [help] check options
+    [q] Quit
+''')
+
+print(help_str)
 
 while True:
-    choice = input()
-
-    if choice.count(" ") > 1:
-        choice = input("\nInvalid input.\n")
-    elif choice.count(" ") == 1:
-        verb, noun = choice.split(" ")
+    command = input("> ").lower().split()
+
+    if len(command) == 1:
+        verb = command[0]
+        if verb == "help":
+            print("\n" + help_str)
+            continue
+        elif verb == "q":
+            print("\nGoodbye!\n")
+            break
+        else:
+            print("Invalid input. Type help for options.\n")
+    elif len(command) == 2:
+        verb, noun = command[0], command[1]
```

Comment:
Nice error handling but if I only type go here I get an error because noun is undefined. Consider adding a condition that prompts the user for a direction here. I do like this approach of separating actions from directions/items it's a pretty neat idea!